test/demo/index.html

```diff
@@ -303,6 +303,11 @@ <h3>Timeseries (secondly)</h3>
           <div class="chart-c3-area-spline-time-stacked keen-chart-container"></div>
         </div>
       </div>
+      <div class="keen-ui-row">
+        <div class="keen-ui-col-4">
+          <div class="chart-metric-combo keen-chart-container"></div>
+        </div>
+      </div>
```

Comment:
at the bottom of the page - create h2 called "combo charts"
let's move this chart there